.github/workflows/test_evaluator.yml

```diff
@@ -0,0 +1,39 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
```

Comment:
you can remove the Github action as all unittests are run by the CircleCI (see `.circleci/config.yml`). there is a bug with it that didn't trigger it on your PR.

Reply:
Yes, I missed that tests were run through CircleCI, I removed the file.